.jenkins/Jenkinsfile-wheel-builder

```diff
@@ -0,0 +1,150 @@
+def configs = [
+    [
+        label: 'windows',
+        versions: ['py26', 'py27', 'py34', 'py35', 'py36'],
```

Comment:
We can do it on Windows but can't do it on Mac (due to the py33 python.org issues we encountered there). I'm kind of inclined to leave py33 out of the wheels entirely and see if anybody complains.